oar/core/notification.py

```diff
@@ -238,6 +237,8 @@ def send_email(self, to_addrs, subject, content):
 
 
 class SlackClient:
+    cache_dict = dict()
```

Comment:
@jhuttana cache object should be moved to class instructor `__init__` like `self.cache = {}`, all subsequent calls need to be changed as well. e.g. `self.cache[xx]=y`

Reply:
That would be perfect. I will do that.